### ISSUE TITLE
Android: 64-bit requirement

### ISSUE THREAD

**Issue by c3bryant** [UCSD/campus-mobile#538]
We received the following warning from Google Play regarding 64-bit requirement

> From August 1, 2019 all releases must be compliant with the Google Play 64-bit requirement.
> 
> Include 64-bit and 32-bit native code in your app. Use the Android App Bundle publishing format to automatically ensure that each device architecture receives only the native code it needs.
> 
> This avoids increasing the overall size of your app. Learn More https://developer.android.com/distribute/best-practices/develop/64-bit

**Comment by c3bryant** [comment]
This will require an update to React Native 0.59+

**Comment by vishant101** [comment]
Found this article https://medium.com/@andriidrozdov/reactnative-and-android-64-bit-new-google-play-market-rules-what-to-do-584b067d6f1a

According to this the only thing we need to do for this to work is upgrade to React-Native 0.59.1+ and check that the android/app/build.gradle file automatically upgrades to include the different architectures for 32 and 64 bit devices. 

I checked the build.gradle file on e68ff3c5e2697632c734da0dabac60bafc43c353 and it seems everything is fine. The last thing we need to do is publish an APK with the React-Native upgrade. Once this is done the warning should disappear.
